app/models/degree/secondary_maths_english_grade4.rb

```diff
@@ -0,0 +1,11 @@
+module Degree
+  class SecondaryMathsEnglishGrade4 < SecondaryMathsEnglishGrade4
+    def next_step
+      if has_required_subjects == "yes"
```

Comment:
Why are theses yes/no instead of a boolean value?

Reply:
Have put some basic tests up, the module will probably change following the content review, as the check answers page has to be changed to a GOV format

Reply:
The results are returned as a hash, I thought 'has_required_subjects' => 'yes', is slightly more readable than a bool.